src/utils.zig

```diff
@@ -1884,3 +1884,70 @@ fn CastFunctionReturnToAnyopaqueType(Fn: type) type {
 pub fn castFunctionReturnToAnyopaque(function: anytype) *const CastFunctionReturnToAnyopaqueType(@TypeOf(function)) {
 	return @ptrCast(&function);
 }
+
+// MARK: Callback
+pub const CallbackError = error{NotFound, EmptyName};
+
+pub fn NamedCallbacks(comptime Child: type, comptime Function: type) type {
+	return struct {
+		const Super = @This();
```

Comment:
Why is this called `Super`?
General convention is to call the current struct `Self` if it cannot be referred to by name.